ground/gcs/src/plugins/config/configoutputwidget.h

```diff
@@ -40,6 +40,33 @@
 class Ui_OutputWidget;
 class OutputChannelForm;
 
+class SignalSingleton : public QObject
+{
+    Q_OBJECT
+public:
```

Comment:
Not sure if this is the best way to have accomplished sharing a signal between the sibling ConfigOutputWidget and ConfigMultirotorWidget.